resources.whatwg.org/build/deploy.sh

```diff
@@ -93,9 +93,11 @@ MATCHES=$(
   perl -ne '$/ = "\n\n"; print "$_" if (/\ban\s+(<[^>]*>)*(?!(L\b|http|https|href|hgroup|rb|rp|rt|rtc|li|xml|svg|svgmatrix|hour|hr|xhtml|xslt|xbl|nntp|mpeg|m[ions]|mtext|merror|h[1-6]|xmlns|xpath|s|x|sgml|huang|srgb|rgb|rsa|rst_stream|only|option|optgroup)\b|html)[b-df-hj-np-tv-z]/si or /\b(?<![<\/;])a\s+(?!<!--grammar-check-override-->)(<[^>]*>)*(?!&gt|one)(?:(L\b|http|https|href|hgroup|rt|rp|li|xml|svg|svgmatrix|hour|hr|xhtml|xslt|xbl|nntp|mpeg|m[ions]|mtext|merror|h[1-6]|xmlns|xpath|s|x|sgml|huang|srgb|rsa|only|option|optgroup)\b|html|[aeio])/si)' "$INPUT_FILE" | perl -lpe 'print "\nPossible grammar problem: \"a\" instead of \"an\" or vice versa (to override, use e.g. \"a <!--grammar-check-override-->apple\"):" if $. == 1'
   grep -ni 'and/or' "$INPUT_FILE" | perl -lpe 'print "\nOccurrences of making Ms2ger unhappy and/or annoyed:" if $. == 1'
   grep -niE '\s+$' "$INPUT_FILE" | perl -lpe 'print "\nTrailing whitespace:" if $. == 1'
+  grep -P '\t' "$INPUT_FILE" | perl -lpe 'print "\nTab:" if $. == 1'
```

Comment:
```suggestion
  grep $'\t' "$INPUT_FILE" | perl -lpe 'print "\nTab:" if $. == 1'
```

Reply:
The `-P` switch is only in GNU grep — it’s not in, for example, the grep that comes in the macOS shell (FreeBSD).

So while we don’t need to worry about that for the case of this particular script — since I guess it’s running in a Linux/GNU shell environment — it’d be better practice to use a more-portable alternative.

The `$'\t'` thing is a Bash-ism but since this script is already explicitly a Bash script — and I guess we have other Bash-isms in it already — then doing `$'\t'` seems preferable.